clang/lib/CodeGen/CGSYCLRuntime.h

```diff
@@ -23,6 +23,19 @@ namespace CodeGen {
 
 class CodeGenModule;
 
+// These aspects are internal and used for device image splitting purposes only.
+// They are not exposed to the DPCPP users through "aspect" enum. That's why
```

Comment:
We're still ~dreaming~ thinking about upstream, let's make it a teeny tiny bit easier.
```suggestion
// They are not exposed to the SYCL users through "aspect" enum. That's why
```

Reply:
Fixed.